### PR TITLE
Add Core-Compute templates

### PR DESCRIPTION
**Pull request by patridge**
Bringing over the new CCM templates from Meadow.SDK, and updating the existing templates as I go.

### CODE REVIEW

File: VS_Meadow_Extension.2019.sln [hunk 1]
```diff
@@ -3,7 +3,7 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.29215.179
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Templates", "Templates", "{F27FC7D7-ADB6-4538-BF59-CBFE72C5AA71}"
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Templates", "Templates", "{470F54A7-8B71-4ECF-94A2-CE167E75963E}"
```
Comment (CartBlanche) [inline VS_Meadow_Extension.2019.sln:6]: @patridge Why was this change required?

Comment (patridge) [reply VS_Meadow_Extension.2019.sln:6]: It wasn't really _required_, but it did align this sln file with the 2022 sln file to make it easier to compare them for future edits.

**Comment by patridge** [comment]
Going to postpone this PR in favor of one that just incorporates the Core-Compute templates: #114. There are changes here that will make new additions easier, and simplify the build configurations. But, as it is, it's not building VS2019 anymore, and it's not worth holding up the new templates.
